File: Pocs/Automated Test/AgileContent/NetCoreStandard/NetCoreSeleniumTest/Pages/MisContenidosPage.cs
using OpenQA.Selenium;

namespace NetCoreSeleniumTest.Pages
{
    public class MisContenidosPage: BasePage
    {
        public By titleContainer;

        public MisContenidosPage(IWebDriver driver): base(driver)
        {
            titleContainer = By.XPath("//div[contains(@class, 'TitleContainer')]/h2[text()='Mi biblioteca']");
        }

        public string GeTtitleContainerText()
        {
            return GetElement(titleContainer).Text;
        }
    }
}

File: Pocs/Automated Test/AgileContent/Node.js/SeleniumMocha/Personal/Pages/mis-contenidos-page.js
const {By} = require('selenium-webdriver');
const BasePage = require('./base-page');

class MisContenidos extends BasePage{

  constructor(driver) {
    super(driver);
    this.titleContainer = By.xpath('//div[contains(@class, "TitleContainer")]/h2[text()="Mi biblioteca"]');    
  }

  async geTtitleContainerText() {
    return await this.getElement(this.titleContainer).getText();
  }
}

module.exports = MisContenidos;